tests/test_model_form_field.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```

Comment:
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.